packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardRenderer.js

```diff
@@ -435,10 +435,13 @@ const AdaptiveCardRenderer = ({ actionPerformedClassName, adaptiveCard, disabled
 
   useEffect(() => {
     // Add aria-pressed and role attribute to the AC action button selected by the user.
-    if (adaptiveCardElementRef.current) {
-      setAttributeWithUndo(adaptiveCardElementRef.current, 'aria-pressed', 'true');
-      setAttributeWithUndo(adaptiveCardElementRef.current, 'role', 'button');
-    }
+    actionsPerformed.map(
+      ({ renderedElement }) =>
+        renderedElement &&
+        adaptiveCardElementRef.current.contains(renderedElement) &&
+        setAttributeWithUndo(renderedElement, 'aria-pressed', 'true') &&
+        setAttributeWithUndo(renderedElement, 'role', 'button')
+    );
```

Comment:
IMO, using if-statement is fine if the statement is more than one line. In your case `setAttributeWithUndo` is called twice. 😉 

```suggestion
    actionsPerformed.forEach(
      ({ renderedElement }) =>
        if (renderedElement && adaptiveCardElementRef.current.contains(renderedElement)) {
          setAttributeWithUndo(renderedElement, 'aria-pressed', 'true');
          setAttributeWithUndo(renderedElement, 'role', 'button');
        }
    );
```